ppr-ui/src/composables/useInputRules.ts

```diff
@@ -2,7 +2,13 @@
 export const useInputRules = () => {
   const maxLength = (maxLength: number): Array<Function> => {
     return [
-      v => v.length <= maxLength || `Maximum ${maxLength} characters`
+      v => (v || '').length <= maxLength || `Maximum ${maxLength} characters`
```

Comment:
What's the difference here? 
Doesn't count spaces?

Reply:
Just preventing throwing a (console) error when `v` is null, so just setting to empty (as per example in the official docs).

Reply:
Ok fair enough, nice work 👍